mobile-app/cypress/integration/functional/wallet/settings/settings.spec.ts

```diff
@@ -48,4 +48,15 @@ context('Wallet - Settings', () => {
     cy.url().should('include', 'app/Settings/AboutScreen')
     cy.getByTestID('app_logo').should('exist')
   })
+
+  it('should activate sun icon in light mode (by default)', () => {
+    cy.getByTestID('light_mode_icon').should('have.css', 'color', 'rgb(251, 191, 36)')
+    cy.getByTestID('dark_mode_icon').should('have.css', 'color', 'rgb(212, 212, 212)')
+  })
+
+  it('should switch and activate moon icon from light to dark mode', () => {
+    cy.getByTestID('light_mode_icon').should('have.css', 'color', 'rgb(251, 191, 36)')
+    cy.getByTestID('theme_switch').click()
+    cy.getByTestID('light_mode_icon').should('have.css', 'color', 'rgb(212, 212, 212)')
```

Comment:
can we add test case for  `Check bottom nav if it changes background and icon colors`

Reply:
Unable to add test the last case on bottom nav as there are no related attributes to use

Reply:
I believe you can add `testID` to `<Materialcons>` like so 
![image](https://user-images.githubusercontent.com/40191153/149776933-73837c92-711b-4ec0-8fb6-bce8ece79529.png)

But to me, a more important test would be to assert whether the `WALLET.THEME` key in local storage is updated